skeleton/package.json

```diff
@@ -5,25 +5,19 @@
   "author": "You",
   "license": "MIT",
   "scripts": {
-    "build": "npm run copy-assets && elm make src/Main.elm --optimize --output=build/app.js && npm run optimize",
-    "copy-assets": "rimraf build && mkdir build && cp -Lr public/* build",
-    "debug": "elm-live src/Main.elm --port=3000 --dir=public/ -- --output=public/app.js --debug",
-    "deploy": "npm run build && gh-pages --dist build/",
-    "live": "elm-live src/Main.elm --port=3000 --dir=public/ -- --output=public/app.js",
-    "optimize": "uglifyjs build/app.js -c $npm_package_uglify_params | uglifyjs -m --output=build/app.js",
```

Comment:
Would be nice to keep/adapt the `deploy` and `optimize` commands.

Reply:
Parcel only has `build` and `serve` (and `watch`) commands. If I understand correctly, parcel passes `--optimize` flag automatically. See https://github.com/parcel-bundler/parcel/issues/2913

Reply:
Maybe just keep the deploy command at least, then ;)